src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const dayRentCost = 40;
+  const rebateOfSevenDays = 50;
+  const rebateOfThreeDays = 20;
```

Comment:
Don't hardcode some values that may changed in the future in variable names.

BAD EXAMPLE:
`const discountForThreeDays = 50;`

GOOD EXAMPLE:
`const basicDiscount = 50;`